.github/workflows/release.yml

```diff
@@ -129,3 +128,19 @@ jobs:
           OSSRH_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
           MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
 
+
+  publish-crates:
+    name: Publish crates.io
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./rust
```

Comment:
This should be enough to get it working though :thinking:

Reply:
@dbaileychess: I think this needs to be `./rust/flatbuffers` and maybe there needs to be a separate one for `./rust/flexbuffers`